### PR TITLE
Add figure 3

### PR DESCRIPTION
**Pull request by hyanwong**
Draft for https://github.com/tskit-dev/what-is-an-arg-paper/issues/36

### CODE REVIEW

**Comment by hyanwong** [comment]
Waiting on #98 for a better graph plot

File: illustrations/pedigree.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
+import json
+from pathlib import Path
+import string
+import sys
+from types import SimpleNamespace
+
+import matplotlib.pyplot as plt
+import tskit
+
+# useful modules in the top level dir
```
Comment (jeromekelleher) [inline illustrations/pedigree.py:10]: You don't need to do this - just run ``python illustrations/pedigree.py`` from the top level dir,

**Review by jeromekelleher — COMMENTED** [review]
This isn't compiling for me - there's no rule to build ``pedigree.pdf``.

Life would be easier if we had a single file ``illustrations.py`` that lived in the top level directory and that produced the different illustrations via subcommands (using click).  Having code mixed in with the output files in ``illustrations`` is making things confusing I think.

File: illustrations/pedigree.py (hunk 1 above)
Comment (hyanwong) [reply illustrations/pedigree.py:10]: right, as long as we remember that's where it's run from, I guess. On the command-line I find it easier to run the things separately from within illustrations.

Comment (jeromekelleher) [reply illustrations/pedigree.py:10]: Let's move all the code for illustrations into one file, ``illustrations.py`` as suggested. Everything gets run from the root, always. Nothing to get confused about then.

**Comment by hyanwong** [comment]
> There should be a pedigree.svg in `illustrations`?

Forgot to git add it, that's all. Sorry. Updated

**Comment by jeromekelleher** [comment]
Works for me now. Look great, thanks. Merge away whenever you're happy.

File: illustrations/pedigree.py (hunk 1 above)
Comment (hyanwong) [reply illustrations/pedigree.py:12]: Oh yes, that's why I did it like this: it's useful to be able to copy and paste the code into a jupyter notebook, as I template and re-test most of the figure code in notebooks which is a lot more responsive than using a script.

I guess I could simply dump the code into scripts and call them within the `illlustrations.py` script. Unless you have better suggestions for easy conversion between notebook cells and free-standing scripts?

Comment (jeromekelleher) [reply illustrations/pedigree.py:12]: Why can't you do that here? You can copy a function definition into a notebook, can't you?

Comment (hyanwong) [reply illustrations/pedigree.py:12]: Oh yes, I guess I could simply turn it into a function. Good point.

**Pull request merged**